content/develop/interact/search-and-query/advanced-concepts/vectors.md

```diff
@@ -9,369 +9,503 @@ categories:
 - oss
 - kubernetes
 - clients
-description: Learn how to use vector fields and vector similarity queries
+description: Learn how to use vector fields and perform vector search in Redis
 linkTitle: Vectors
 math: true
 title: Vectors
 weight: 14
 ---
 
-*Vector fields* allow you to use vector similarity queries in the [`FT.SEARCH`]({{< baseurl >}}/commands/ft.search/) command.
-*Vector similarity* enables you to load, index, and query vectors stored as fields in Redis hashes or in JSON documents (via integration with the [JSON module]({{< relref "/develop/data-types/json/" >}}))
+Redis is commonly used as a [highly-performant vector database](https://redis.io/blog/benchmarking-results-for-vector-databases/). Vector fields enable you to perform semantic search based on vector embeddings in combination with metadata like text, numerics, geo, or tags.
 
-Vector similarity provides these functionalities:
+**Just looking to get started?** Checkout the vector [quickstart guide]({{< baseurl >}}/develop/get-started/vector-database) and the [Redis AI Resources](https://github.com/redis-developer/redis-ai-resources) repo for more assistance.
 
-* Realtime vector indexing supporting two indexing methods:
 
-    - FLAT - a simple, lightweight index.
+## Overview
 
-    - HNSW - Modified version of [nmslib/hnswlib](https://github.com/nmslib/hnswlib), which is an implementation of [Efficient and robust approximate nearest neighbor search using Hierarchical Navigable Small World graphs](https://arxiv.org/ftp/arxiv/papers/1603/1603.09320.pdf).
+### Storage options
```

Comment:
It goes from a very brief overview to realtime indexing without a lot of context here. 

I think adding a brief sentence or two outlining how the pieces work together to enable vector search could be really valuable to level-set.

- redis stores vectors as a field in an object of type hash or json
- to query for vectors you need to create an index for the object and for the vector
- both these indexes come with certain configurations

Generally I think we need something here to show how the piece come together in the end a bit more if that makes sense

Reply:
Agreed. Updated this @rbs333